src/main/groovy/io/openliberty/tools/gradle/Liberty.groovy

```diff
@@ -164,8 +165,19 @@ class Liberty implements Plugin<Project> {
            } else {
                return new File(project.liberty.baseDir, 'wlp')
            }
-        } else {
-           return new File(project.liberty.installDir)
+        } else { // installDir is specified
+            File installDir
+            if (project.liberty.installDir.endsWith("wlp")) {
+                installDir = new File(project.liberty.installDir)
+            } else { // not valid wlp dir
+                project.getLogger().warn(MessageFormat.format("The installDir {0} is not pointing to a wlp folder. Trying {0}/wlp", project.liberty.installDir))
```

Comment:
The installDir {0} path does not reference a wlp folder. Checking for a valid Liberty installation at path {0}/wlp instead.